res/database/Default/user/scr0llbaer.xml

```diff
@@ -419,30 +418,75 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="d6242a49-71e0-4997-97d6-3684e5c67d54" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="d6242a49-71e0-4997-97d6-3684e5c67d54" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+			<title>
+				<de>%tower% Dynamit - Das Magazin: %latebreaking%</de>
+				<en>%tower% Dynamite - The Magazine: %latebreaking%</en>
+			</title>
+			<description>
+				<de>Showbiz aktuell. Jeden Tag neu. Aktueller Tratsch, Promi-Geflüster und tränenrührende Tagesgeschehnisse.
+Ein Muss für jede Hausfrau, ein Fluch für harte Männer.</de>
+				<en>Showbiz news. Updated every day. Current gossip, celebrity rumors and tear-jerking daily events.
+A must for every housewife, a curse for hard men.</en>
+			</description>
+
+			<variables>
+				<tower>
+					<de>TVT|Tower|TVTower</de>
+					<en>TVT|Tower|TVTower</en>
+				</tower>
+				<latebreaking>
+					<de>Aktuell|Brandaktuell|Brandheiße News|Aktuellste News</de>
+					<en>Late-Breaking|The Hot Stuff|Blazing News|Breaking News</en>
+				</latebreaking>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="64" required="0" />
+			</jobs>
+			<data flags="17" production_limit="100" production_broadcast_limit="2" production_broadcast_flags="512" production_licence_flags="36" />
+			<genres maingenre="301" subgenres="204,7" />
+			<blocks value="1" />
+			<price min="10000" max="20000" />
+			<potential min="25" max="55" />
+			<outcome min="25" max="55" />
+			<review min="25" max="55" />
+			<speed min="25" max="55" />
+			<production_time value="60" />
+			<programme_data_modifiers>
+				<modifier name="topicality::age" value="5" />
+				<modifier name="topicality::refresh" value="2" />
+			</programme_data_modifiers>
+		</scripttemplate>
+
+		<scripttemplate product="3" licence_type="1" guid="007b49ae-41e2-47b7-bf7c-628aecee7731" creator="9551">
```

Comment:
Topic-wise this looks a lot like TheRob-Script-Talk-DieNeuestenFilme, but this is more a deluxe trash version(?)

Reply:
indeed copy-paste-error, was not intending it to be trash.

Yeah maybe similar, but I think it's not a problem. Movies are such a central element of the game.